samples/observability.js

```diff
@@ -0,0 +1,284 @@
+/*!
+ * Copyright 2024 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// sample-metadata:
+//   title: Observability (Tracing) with OpenTelemetry
+//   usage: node observability.js trace <INSTANCE> <DATABASE> <PROJECT-ID>
+
+'use strict';
+
+// Setup OpenTelemetry and the trace exporter.
+// [START spanner_trace_and_export_spans]
+const {Resource} = require('@opentelemetry/resources');
+const {NodeSDK} = require('@opentelemetry/sdk-node');
+const {
+  NodeTracerProvider,
+  TraceIdRatioBasedSampler,
+} = require('@opentelemetry/sdk-trace-node');
+const {
+  BatchSpanProcessor,
+  ConsoleSpanExporter,
+  SimpleSpanProcessor,
+} = require('@opentelemetry/sdk-trace-base');
+const {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMRESATTRS_SERVICE_VERSION,
+} = require('@opentelemetry/semantic-conventions');
+
+const resource = Resource.default().merge(
+  new Resource({
+    [SEMRESATTRS_SERVICE_NAME]: 'spanner-sample',
+    [SEMRESATTRS_SERVICE_VERSION]: 'v1.0.0', // The version of your app running.,
+  })
+);
+
+// Create the Google Cloud Trace exporter for OpenTelemetry.
+const {
+  TraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+const exporter = new TraceExporter();
+
+// Optionally, you can enable gRPC instrumentation.
+if (process.env.ENABLE_GRPC_TRACING === 'true') {
+  const {registerInstrumentations} = require('@opentelemetry/instrumentation');
+  const {GrpcInstrumentation} = require('@opentelemetry/instrumentation-grpc');
+  registerInstrumentations({
+    instrumentations: [new GrpcInstrumentation()],
+  });
+}
+
+function traceAndExportSpans(instanceId, databaseId, projectId) {
+  // Wire up the OpenTelemetry SDK instance with the exporter and sampler.
+  const sdk = new NodeSDK({
+    resource: resource,
+    traceExporter: exporter,
+    // Trace every single request to ensure that we generate
+    // enough traffic for proper examination of traces.
+    sampler: new TraceIdRatioBasedSampler(1.0),
+  });
+  sdk.start();
+
+  // Create the tracerProvider that the exporter shall be attached to.
+  const provider = new NodeTracerProvider({resource: resource});
+  provider.addSpanProcessor(new BatchSpanProcessor(exporter));
+
+  // This makes a global tracerProvider but you could optionally
+  // instead pass in the provider while creating the Spanner client.
+  provider.register();
```

Comment:
This should be commented. And update comment as 
``Un-comment below line to used global tracerProvider instead of passing via Spanner Options`

Reply:
Done, thanks!